README.md

```diff
@@ -92,6 +92,24 @@ export class MyComponent implements OnInit {
 }
 ```
 
+## How to change service url base
+
+The base url of the service are extracted from de swagger file, but in case of needed it can be changed.
```

Comment:
Please, change to _The base url of the API is extracted from the swagger file, but in case of need, it can be changed._

Reply:
Sorry, I am not a native.

Reply:
Done